src/components/prediction/PredictionPage.tsx

```diff
@@ -79,10 +98,20 @@ const PredictionPage = () => {
           temperatureData={temperatureData}
           precipitationData={precipitationData}
           period={period}
+          setErrorMessages={setErrorMessages}
           orgUnits={orgUnits}
-          orgUnitLevels={orgUnitLevels}
+          orgUnitLevel={orgUnitLevel}
         />
       )}
+      {errorMessages.map((error : ErrorResponse, index) => (
+        <div key={index} className={styles.errorBar}>
+          <div className={styles.errorHeader}>
+            <IconError24/>
+            <span>{error.element} request failed</span>
+          </div>
+            <span className={styles.detailedError}>{JSON.stringify(error.error, null, 2)}</span>
+        </div>
+      ))}
```

Comment:
This looks like this
![image](https://github.com/dhis2/chap-app/assets/49228830/e4f91133-9f00-401b-8783-18c6da6106fd)